airbyte-integrations/connectors/destination-kafka/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 }
 
 application {
-    mainClass = 'io.airbyte.integrations.destination.kafka.KafkaDestination'
+    mainClass = 'io.airbyte.integrations.destination.kafka.KafkaDestinationRunner'
```

Comment:
@tuliren is this the right usage? I.e.:
1. create a new StrictEncryptDestination class to wrap the original Destination class
2. create a new Runner wrapper class to call either the old or new Destinations classes
3. switch this mainClass to call the new Runner class

Reply:
Yes! Nice.

Reply:
You can test it by:
1. Launch a normal local instance, and the non-strict encrypt spec should show up.
2. Add `DEPLOYMENT_MODE=CLOUD` in the `.env` file and launch the local instance again, only the strict encrypt spec should show up.

Based on my experience, one tricky thing is that only the `docker-compose` command will respect the `.env` file. The newer `docker compose` will not.

Reply:
fyi you can also just pass the env var on the command line, e.g. `DEPLOYMENT_MODE=CLOUD VERSION=dev docker-compose up`. In my testing it also works with the newer `docker compose` :)

Reply:
Cool. That's good to know.